### DISCUSSION TITLE
Обновление версии под Firefox

### DISCUSSION THREAD

**Discussion by karmicdude** [Chimildic/YaMuTools#48] [Вопрос/ответ] [answered]
В https://addons.mozilla.org/ru/firefox/addon/yamutools/ старая версия. Можно обновить или краткую инструкцию по сборке?

**Comment by Chimildic** [comment, marked as answer]
Обновил для Firefox. Релиз доступен в [аддонах](https://addons.mozilla.org/ru/firefox/addon/yamutools/) и [github](https://github.com/Chimildic/YaMuTools/releases/tag/v0.8.0).

**Reply by kisoid** [reply]
Будет ли обновление для FF до 0.8.7? Там сейчас последняя версия аж 0.8.1

**Reply by Chimildic** [reply]
@kisoid Портировать не так интересно, как делать новое) Шучу. Подожду пока, на случай ошибок/правок.

**Comment by kisoid** [comment]
Что случилось с версией для FF?

Поставил себе Ubuntu 24.04 + FF 127
Зашёл на страницу аадона, а там: "Ой! Мы не можем найти эту страницу"

Решил проверить под Win7 + FF 102 ESR
Такое же сообщение на странице: "Ой! Мы не можем найти эту страницу"
При этом в FF на Win7 у меня расширение установлено и обновлено до версии 0.8.11 (Последнее обновление 23 июня 2023 г.)
При этом на гитхабе последняя версия 0.8.10 от Dec 15, 2022 в релизах... (??)

**Reply by Chimildic** [reply]
Расширение долгое время без поддержки. Некоторые функции отвалились из-за изменений на сайте. Например, импорт из ластфм не работает потому что яндекс отключил старый импорт. Поэтому для фокса оно скрыто, а для хрома работает с багами.

По хорошему его надо переписывать с новым опытом. Но я столько раз уже делал что-то связаное с музыкой, что голового энтузиазма уже не хватает. Возможно когда нибудь вернусь.

Если очень нужно под фокс - установите через режим разработчика версию с гитхаба.

**Reply by kisoid** [reply]
Так и сделал в итоге - установил через "Временные расширения":
about:addons -> about:debugging#/runtime/this-firefox

Но неудобно... работает только до перезагрузки браузера

Ещё откопал на компе предусмотрительно сохранённый в своё время xpi-файл для версии 0.8.1
Он нормально добавляется на постоянку.
ZIP на постоянку не добавляется. Пишет "corrupt".

Нет ли возможности добавить в раздел релизов на гитхабе xpi для 0.8.11 или 0.8.10?
Чтобы расширение не слетало после перезагрузки...
